letsdrill.rb

```diff
@@ -1,12 +1,29 @@
 def get_letter_grade(integer)
  
   #Put your code here!
+  case integer
+  when 90..100
+    return "A"
+  when 80...90
+    return "B"
+  when 70...80
+    return "C"
+  when 60...70
+    return "D"
+  else
+    return "F"
+  end
+  
 
 end
 
 def shortest_string(array)
 
   #Put your code here!
+  sorted_array = array.sort_by(&:length)
```

Comment:
You're almost there. `&` is actually `to_proc` so what you're getting is a fancy lambda - a first class function. You'll see this shorthand a lot in rails production code. Feel free to look up the difference between procs, blocks & lambdas in ruby.

You see similar things in js & python.

python
```python
map(lambda x: x+2, [1,2])
```

js
```js
[1,2].map(function(x){ return x + 2; })
```

ruby
```ruby
[1,2].map(&->(x){x*2})
[1,2].map { |x| x+2 }
```